src/documentation/nl_syntus_fares_read.md

```diff
@@ -0,0 +1,57 @@
+# Syntus fares reading specification
+## Introduction
+This document describes how fares provided by Keolis Netherlands are read in Navitia Transit Model (NTM).
+
+## Input data description
+The fare data expected at the input of the connector comes in the form of XML files based on [Netex fare exchange format](http://www.normes-donnees-tc.org/wp-content/uploads/2014/07/BNTRA-CN03-GT7_N0064_prCEN_TS_278330_FV_E-part_3-v9-1.pdf). Each file is composed of 3 basic elements:
+- a *ResourceFrame* specifying the data owner and the validity period of the provided data
+- a *ServiceFrame* specifying the network, the lines and the stops to which the fare structure is applied
+- one or more *FareFrame*s specifying the fare structures and the underlying conditions.
+
+The supported fare structures depend on origin-destination (OD) stop pairs in two ways:
+- the ticket price beteween an origin and a destination is directly specified (*DirectPriceMatrix*)
+- the price for a fare distance unit is specified, as well as the fare distance between an origin and destination (*UnitPrice* & *DirectMatrix*). The ticket price between the origin and destination stops is computed by multiplying the fare distance by the fare distance unit.
```

Comment:
"between `an` origin and destination" >" between origin and destination" or "between an origin and `a` destination"